app/controllers/application_controller.rb

```diff
@@ -5,8 +5,8 @@ class ApplicationController < ActionController::Base
   include Pundit
 
   # Pundit: white-list approach.
-  after_action :verify_authorized, except: :index, unless: :skip_pundit?
-  after_action :verify_policy_scoped, only: :index, unless: :skip_pundit?
+  # after_action :verify_authorized, except: :index, unless: :skip_pundit?
```

Comment:
And maybe I would uncomment the pundit so that we don't interfere with Shante's work? :) (I understand you needed to do that to access the home page)

Reply:
Yeah I needed to temporarily comment it haha, but if @TofuSupreme made a PR for this we can uncomment it no problem

Reply:
Okey dokey! You’re right when we pull her request that should be back to normal :)